app.js

```diff
@@ -41,6 +41,12 @@ app.get('/crit/:crit_id', (req, res) => {
 	});
 	
 });
+//welcome page route
+app.get('/Welcome', (req, res) => {
+	res.render('NEWUSER');})
+        //route for profile

```

Comment:
This comment should be lined up with the start of the line. Also, add an extra line between these two routes.